app/config/application.yml

```diff
@@ -0,0 +1,8 @@
+logging:
+    level:
+        be.xplore.githubmetrics: DEBUG
+management:
+    endpoints:
+        web:
+            exposure:
+                include: '*'
```

Comment:
This will also enable and expose the shutdown endpoint and various other sensitive endpoints to everyone.  Only expose the endpoints you need

Reply:
fixed